pyrovision/models/pyronear_model.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+from pyrovision.models.utils import cnn_model
+import holocron
+from holocron.models.utils import load_pretrained_params
+
+
+url = "https://github.com/pyronear/pyro-vision/releases/download/v0.1.0/rexnet1_0x_acp_2e017f83.pth"
```

Comment:
I suggest replicating this: https://github.com/frgfm/Holocron/blob/master/holocron/models/rexnet.py#L13-L24

Here we create only a dict, with the function as a key (rexnet1_0x) and the url as a value. That way if we add another rexnet, we'll just have to add an entry.